File: exs/mundo_1/javascript/024.js
/*
Desafio 024

Problema: Crie um programa que leia o nome de uma cidade diga se ela começa ou não com o nome "SANTO".

Resolução do problema:
*/

const input = require('readline-sync').question //Chamando o modulo readline-sync para entrada de dados

//Entrada de dados
const cidade = input('Digite o nome da cidade: '.toLowerCase())
console.log(cidade.indexOf('santo') != -1 ) // Função indexOf é para saber se palavra que voce quer esta na frase.

File: exs/mundo_1/rust/024.rs
/*
Desafio 024

Problema: Crie um programa que leia o nome de uma cidade diga se ela começa ou não
          com o nome "SANTO".
*/

fn main() {
    let mut input = String::new();

    println!("Digite o nome da cidade: ");
    std::io::stdin().read_line(&mut input).unwrap();

    let inicia_com_santo = input.trim().to_uppercase().starts_with("SANTO");

    println!("O nome da cidade digitada começa com \"SANTO\": {inicia_com_santo}");
}
